consumer_group_test.go

```diff
@@ -3,8 +3,50 @@ package sarama
 import (
 	"context"
 	"fmt"
+	"testing"
 )
 
+func TestConsumerGroupSessionNextOffset(t *testing.T) {
+	var partition int32 = 13
```

Comment:
I like using variable blocks like this, but it's a personal preference.
```suggestion
	var (
	    partition int32 = 13
	    offset int64 = 23
	    meta = "meta"
```